odoo/service/server.py

```diff
@@ -394,7 +394,8 @@ def signal_handler(self, sig, frame):
             os._exit(0)
         elif sig == signal.SIGHUP:
             # restart on kill -HUP
-            odoo.phoenix = True
+            global server_phoenix  # noqa: PLW0603
```

Comment:
I don't agree with this noqa, a more elegant solution would have been to set a default value to False to avoid the getattr. 

I understand that your goal is to remove a variable on odoo to be able to make it a namespace.

You will have the same issue with odoo.evented, it may make sense to find an elegant way to store that, maybe having a config object on the server with different flags. I don't say that it is the right solution, but I'm not a big fan of using global ;)

Reply:
The variable is used to restart the process, so it should not be an issue to have it as a global variable. By the way, the config is also a kind of global variable in Odoo too :)
Since `phoenix` is used only here, I just moved it to this module.

Indeed, for 'evented' it would be similar, but I started with the simpler case.